syntax_and_semantics/literals/string.md

```diff
@@ -24,15 +24,15 @@ Available escape sequences:
 "\r"                  # carriage return
 "\t"                  # tab
 "\v"                  # vertical tab
-"\888"                # octal ASCII character
+"\377"                # octal ASCII character
```

Comment:
I'd rather not use the max value here. `\x888` is supposed to illustrate it's base 8. That's probably fine.

Reply:
`\888` does not represent an octal number. Max would be `\777`, but even this value is misleading, if in this context the language accepts \377 as the max